jsk_perception/sample/sample_darknet_yolo_object_detection.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
```

Comment:
?? Sorry, I cannot understand what you mean. You don't like filename? If so please rename as you like.